packages/core/src/llm/LLM.ts

```diff
@@ -635,6 +632,7 @@ export class Anthropic extends BaseLLM {
   maxRetries: number;
   timeout?: number;
   session: AnthropicSession;
+  systemPrompt?: string;
```

Comment:
We want to maintain the same API for other LLMs for systemPrompt, so we'll need to rejigger the logic for Anthropic.